ow_lander/scripts/camera_capture_action_server.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+
+import actionlib
+
+import ow_lander.msg
+from std_msgs.msg import Empty, Float64
+from sensor_msgs.msg import PointCloud2
+
+class CameraCaptureActionServer(object):
+  #_result = std_msgs.msg.Bool()
+
+  def __init__(self, name):
+    self._action_name = name
+    self._result = ow_lander.msg.CameraCaptureResult()
+    self._exposure_pub = rospy.Publisher(
+      '/gazebo/plugins/camera_sim/exposure', Float64, queue_size=10)
```

Comment:
Just curious: RQT also exposes `/gazebo/plugins/camera_sim/gui/exposure` and `/gazebo/plugins/camera_sim/gui/gamma`.  What are these for, and would they be of interest in this action?

Reply:
That camera doesn't apply here. The "gui" camera is the one used to view the scene in gzclient. It renders frames constantly and is not triggerable.